data/com.endlessm.SystemExternalApps.rules.in

```diff
@@ -0,0 +1,17 @@
+polkit.addRule(function(action, subject) {
+    if (action.id == "com.endlessm.SystemExternalApps.build-install" &&
+        action.lookup("user") == "@helper_user@" && subject.active == true &&
+	subject.local == true) {
+	    if (subject.isInGroup("@privileged_group@"))
+	        return polkit.Result.YES;
+	    else
+	        return polkit.Result.AUTH_ADMIN;
+    }
+    if ((action.id == "org.freedesktop.Flatpak.app-install" ||
+         action.id == "org.freedesktop.Flatpak.runtime-install" ||
+	 action.id == "org.freedesktop.Flatpak.runtime-uninstall") &&
```

Comment:
You're right about the app-install. The runtime-uninstall was thinking of also removing the runtime extensions without prompting for the password to avoid having this done twice (once for the app, another for the runtime extension) but it's not being used ATM so I will remove it and add it once we have another script for removing a runtime extension.